sql/moz-fx-data-shared-prod/fivetran_costs_derived/daily_connector_costs_v1/checks.sql

```diff
@@ -1,5 +1,4 @@
 {{ is_unique(columns=["destination", "measured_date", "connector", "billing_type"]) }}
+ {{ not_null(columns=["destination", "measured_date", "connector", "billing_type", "active_rows", "cost_in_usd"]) }}
+ {{ min_rows(1) }}
```

Comment:
The `bqetl format` formatting of macros in these `checks.sql` files isn't the best, with the leading space on subsequent lines.  Maybe that can be addressed in a separate PR.

I thought perhaps adding trailing semicolons after each macro would work, and that does result in nicer `bqetl format` formatting here, but when the macros are expanded they include a trailing semicolon of their own, and BigQuery doesn't seem to handle consecutive semicolons gracefully, resulting in a `Syntax error: Expected end of input but got ";"`.

Reply:
We could remove the `;` from the macro definition here and instead require the user to include it at the end of the line when referencing each macro. This might actually be more intuitive since if custom sql check is written inside the checks.sql file then a semicolon will be required at the end of each custom sql.